be/src/olap/rowset/segment_v2/inverted_index_file_writer.cpp

```diff
@@ -282,6 +282,10 @@ int64_t InvertedIndexFileWriter::write_v1() {
             ram_dir.deleteFile(idx_name.c_str());
             ram_dir.close();
 
+            if (is_index_compaction && config::dual_write_inverted_index_enable) {
```

Comment:
DO NOT couple debug code with the main process logic.

Reply:
Done.